libparsec/crates/platform_mountpoint/tests/fuse.rs

```diff
@@ -0,0 +1,25 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+#[cfg(target_os = "linux")]
+use libparsec_tests_fixtures::prelude::*;
+
+#[cfg(target_os = "linux")]
+#[parsec_test]
+fn unmount_with_fusermount(tmp_path: TmpPath) {
+    let path = &*tmp_path;
+    let path = path.join("mountpoint");
+    let bar = path.join("bar");
+
+    libparsec_platform_mountpoint::mount(&path).unwrap();
+
+    std::fs::create_dir(&bar).unwrap();
+
+    assert!(std::process::Command::new("fusermount")
```

Comment:
Can you check that `bar` exist before the command ?

Reply:
Do you mean, we can't trust `std` ? xD

Reply:
:x yeah we can trust `std`